app/views/content/home/_calls-to-action.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= render CallsToAction::HomepageComponent.new(
     icon: "icon-person",
     title: "Get free one-to-one support",
-    text: "Our expert advisers have years of teaching experience. They offer practical advice as well as reassurance. Chat by phone, email, text or WhatsApp.",
+    text: "Our expert advisers offer free, one-to-one support, whether you’re just thinking about teaching or ready to apply. Chat by phone, email, text or WhatsApp.",
```

Comment:
Given we say `one-to-one` in the title, is it worth saying something different here? But happy for it to remain as is / you to change my suggestion which I'm not sure is quite right!

```suggestion
    text: "Our expert advisers offer free individual support, whether you’re just thinking about teaching or ready to apply. Chat by phone, email, text or WhatsApp.",
```